x-pack/plugins/reporting/server/lib/tasks/execute_report.ts

```diff
@@ -337,6 +370,32 @@ export class ExecuteReportTask implements ReportingTask {
     this.logger.info(`Saved ${report.jobtype} job ${docId}`);
     report._seq_no = resp._seq_no;
     report._primary_term = resp._primary_term;
+
+    // event tracking
+    const startedAt = report.started_at;
+    const eventTracker = this.getEventTracker(report);
+    if (docOutput.metrics?.csv != null) {
+      eventTracker?.completeJobCsv({
+        byteSize: docOutput.size,
+        timeSinceClaimed: startedAt
+          ? completedTime.valueOf() - new Date(startedAt).valueOf()
+          : undefined,
+        csvRows: docOutput.metrics.csv.rows ?? 0,
+        // csvColumns: docOutput.metrics?.csv?.columns, // FIXME: doesn't exist yet
+      });
+    } else if (docOutput.metrics?.pdf != null || docOutput.metrics?.png != null) {
+      eventTracker?.completeJobScreenshot({
+        byteSize: docOutput.size,
+        timeSinceClaimed: startedAt
+          ? completedTime.valueOf() - new Date(startedAt).valueOf()
+          : undefined,
+        numPages: docOutput.metrics.pdf?.pages ?? 1,
+        screenshotLayout: report.payload.layout?.id ?? 'preserve_layout',
+        // screenshotPixels: docOutput.metrics?.screenshot?.pixels, // FIXME: doesn't exist yet
```

Comment:
I propose adding this part in a separate PR, since we'll want to create new info under `ReportOutput['metrics']` for it.